core-ui/src/components/Predefined/Create/ApiRules/AccessStrategyForm.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ResourceForm } from 'shared/ResourceForm/ResourceForm';
+import * as Inputs from 'shared/ResourceForm/components/Inputs';
+import { JwtStrategyConfig } from './JwtStrategyConfig';
+import { createAccessStrategyTemplate } from './templates';
+
+const accessStrategyOptions = [
+  {
+    key: 'allow',
+    text: 'Allow',
```

Comment:
shouldn't these be passed through i18n? i know they're simple cases, and most probably will be identical regardless of language... but since the text value is prettified for human readability, it feel it should be in translation files, not here.
(then ``['allow', 'noop'].map(key => ({ key, text: t(`blah.blah.${key}`) }))`` could be used instead)